pynecone/app.py

```diff
@@ -148,7 +148,7 @@ def add_cors(self):
             allow_origins=["*"],
         )
 
-    def preprocess(self, state: State, event: Event) -> Optional[Delta]:
+    async def preprocess(self, state: State, event: Event) -> Optional[Delta]:
```

Comment:
Should we make `postprocess` async also for consistency?

Reply:
yep, I agree